src/hb-bit-page.hh

```diff
@@ -86,6 +86,26 @@ struct hb_bit_page_t
   void set_range (hb_codepoint_t a, hb_codepoint_t b, bool v)
   { if (v) add_range (a, b); else del_range (a, b); }
 
+  // Writes out page values to the array p. Returns a pointer to where the next
+  // value should be written, after the values we wrote (if any).
+  // At most size codepoints will be written. Returns the number of codepoints written.
+  unsigned int export_array (uint32_t base, hb_codepoint_t *p, unsigned int size) const
+  {
+    unsigned int count = 0;
+    for (unsigned i = 0; i < len () && count < size; i++)
+    {
+      elt_t bits = v[i];
+      uint32_t v_offset = i << ELT_BITS_LG_2;
```

Comment:
Since this is the only use of `ELT_BITS_LG_2` you could as well just multiply by `ELT_BITS`. That would avoid hardcoding sizeof `elt_t` as you have essentially done now.